modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/client/AsahFaroBackendClientImpl.java

```diff
@@ -88,16 +93,12 @@ public Long calculateExperimentEstimatedDaysDuration(
 		ExperimentSettings experimentSettings) {
 
 		try {
-			AnalyticsConfiguration analyticsConfiguration =
-				_analyticsSettingsManager.getAnalyticsConfiguration(companyId);
-
-			String days = _jsonWebServiceClient.doPost(
-				String.class,
-				analyticsConfiguration.liferayAnalyticsFaroBackendURL(),
+			String days = _getResponse(
+				companyId, Http.Method.POST, null,
```

Comment:
It is not a good practice to send null to a private method, as we need to handle null values, what do you think about use  Collection.emptyMap?